dataprofiler/tests/profilers/test_data_labeler_column_profile.py

```diff
@@ -241,3 +242,29 @@ def test_profile_merge(self, mock_instance):
         with self.assertRaises(ValueError):
             profiler._top_k_voting = 13
             test = profiler + profiler2
+            
+    def test_data_labeler_column_with_wrong_options(self, *mocks):
+        with self.assertRaisesRegex(ValueError,
+                                   "options must be of type DataLabelerOptions."):
+            profiler = DataLabelerColumn("Data_Labeler", options="wrong_data_type")
+
+    def test_profile_merge(self, mock_instance):
+        self._setup_data_labeler_mock(mock_instance)
+
+        data = pd.Series(['1', '2', '3', '11'])
+        data2 = pd.Series(['4', '5', '6', '7', '9', '10', '12'])
+        options = DataLabelerOptions()
+        options.max_sample_size = 20
+        profiler = DataLabelerColumn(data.name, options=options)
+        profiler.update(data)
+
+        options2 = DataLabelerOptions()
```

Comment:
can we add a test for two data labelers with different options?

Reply:
Done!